tests/PHPUnit/Util/Log/VCRTest.php

```diff
@@ -23,4 +14,39 @@ public function testInterceptsWithAnnotations()
         $result = file_get_contents('http://google.com');
         $this->assertEquals('This is a annotation test dummy.', $result, 'Call was not intercepted (using annotations).');
     }
-}
\ No newline at end of file
+
+    /**
+     * @vcr unittest_annotation_test.yml
+     */
+    public function testInterceptsWithAnnotationsAndFileExtension()
+    {
+        $result = file_get_contents('http://google.com');
+        $this->assertEquals('This is a annotation test dummy.', $result, 'Call was not intercepted (using annotations).');
+    }
+
+    /**
+     * @vcr unittest_annotation_test_auto_generated.yml
+     */
+    public function testInterceptsWithAnnotationsAutoGenerated()
```

Comment:
Not sure if it needs a test because this is intensively tested within php-vcr. 
But keeping it is ok, I just wonder if we don't delete the fixture file in the test or use [vfsStream](https://github.com/mikey179/vfsStream) that we can get false-positives.